tests/prague/eip7692_eof_v1/eip7069_extcall/test_calls.py

```diff
@@ -796,22 +814,27 @@ def test_eof_calls_min_callee_gas(
     )
 
 
-def test_eof_calls_no_balance(
+@pytest.mark.parametrize("balance", [0, 1, 2, 2**256 - 1])
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "balance", [0, 1, 2, pytest.param(2**256 - 1, marks=pytest.mark.pre_alloc_modify)]
)
```
Adding a marker just to make sure we don't try to run this test in live networks due to the required balance amount.